docker/Dockerfile.aardvark-base

```diff
@@ -0,0 +1,38 @@
+
+FROM python:2
+
+LABEL author="scottbrown0001@gmail.com"
+
+COPY phantomjs/bin/phantomjs /usr/local/bin
+
+COPY aardvark /tmp/aardvark
+WORKDIR /tmp/aardvark
+
+# # Update the repository and install tools and aardvark.
+# RUN apt-get update && \
+# 	apt-get install -y vim && \
```

Comment:
do we need vim?

Reply:
Good catch. Not needed for anything in the build-run path. I also see I left the update step commented out by mistake - doubly good catch. Will commit an update in a sec.

Reply:
Ok, that was a long sec. Re-ran the test file and found an issue with how the AARDVARK_IMAGES_TAG env variable was handled, fixed that too and took the time to clean up some highly repetitive code, for posterity.